plugin.js

```diff
@@ -37,14 +41,14 @@ Plugin.registerSourceHandler("less", {archMatching: 'web'}, function (compileSte
   if ( config.useIndex ) {
     var indexFilePath = config.indexFilePath || DEFAULT_INDEX_FILE_PATH;
     // If this isn't the index file, add it to the index if need be
-    if ( compileStep.inputPath != indexFilePath ) {
+    if ( convertToPosixPath(compileStep.inputPath) != indexFilePath ) {
       if ( fs.existsSync(indexFilePath) ) {
         var lessIndex = fs.readFileSync(indexFilePath, 'utf8');
-        if ( lessIndex.indexOf(compileStep.inputPath) == -1 ) {
-          fs.appendFileSync(indexFilePath, '\n@import "' + compileStep.inputPath + '";', 'utf8');
+        if ( lessIndex.indexOf(convertToPosixPath(compileStep.inputPath)) == -1 ) {
+          fs.appendFileSync(indexFilePath, '\n@import "' + convertToPosixPath(compileStep.inputPath) + '";', 'utf8');
         }
       } else {
-        var newFile = generatedMessage + '@import "' + compileStep.inputPath + '";\n';
+        var newFile = generatedMessage + '@import "' + convertToPosixPath(compileStep.inputPath) + '";\n';
         fs.writeFileSync(indexFilePath, newFile, 'utf8');
```

Comment:
No problem,

* Kind of platform independent would be posix style which i think works on windows to (i am currently on the mac so cant do a quick test). And secondly i think it should comply with the less specification, which is using url style paths
* because of the mentioned [specification](http://lesscss.org/features/#variables-feature-import-statements) of the less paths the compiler works fine on windows

Only converting the `indexFilePath` will not be enough, but storing the converted `compileStep.inputPath` would be better that is true i will update the pull request accordingly.